<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-RO8P4O0" failures="1" tests="4" name="Test" time="69.283" errors="0" timestamp="2022-03-24T16:03:44 IST">
  <testcase classname="Securelandlogin.login" name="adminlogin" time="10.103"/>
  <testcase classname="Securelandlogin.login" name="empLogin" time="16.011"/>
  <testcase classname="Securelandlogin.login" name="createUser" time="27.096">
    <failure type="java.lang.AssertionError" message="expected [http://localhost/SecureUrlandz/register] but found [http://localhost/SecureUrlandz/]">
      <![CDATA[java.lang.AssertionError: expected [http://localhost/SecureUrlandz/register] but found [http://localhost/SecureUrlandz/]
at Securelandlogin.login.createUser(login.java:79)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- createUser -->
  <testcase classname="Securelandlogin.login" name="createUser2" time="14.474"/>
</testsuite> <!-- Test -->
